modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -147,35 +144,35 @@ If your secrets group does not appear, ensure you have the *Manage Runtime Fabri
 .. From the *TLS Context* menu, select the TLS Context to be used for your Runtime Fabric.
 
 
-. Click *Deploy* to deploy Runtime Fabric. If successful, a message appears in the bottom-right of the page. The deployment may take up to a minute. You can view the deployment status at top of the form. When the status transitions to *Applied*, the internal load balancer is successfully deployed and inbound traffic enabled.
+. Click *Deploy* to deploy the internal load balancer. If successful, a message appears in the bottom-right of the page. The deployment may take up to a minute. You can view the deployment status at thw top of the form. When the status transitions to *Applied*, the internal load balancer is successfully deployed and inbound traffic enabled.
 
 == Verify that Inbound Traffic is Enabled
 
 To resolve the Common Name (CN) of applications deployed on Runtime Fabric, DNS must be configured to map the CN to the IP address of the external load balancer or of each controller VM. To test inbound traffic for applications before configuring DNS, send a request to the application using the IP address and a host header set to the domain.
 
-The structure of the domain depends on whether a wildcard was used in your certificate, for example:
+The structure of the domain depends on whether a wildcard was used in the CN of your certificate, for example:
 
-* A CN with a wildcard (e.g. *.example.com) uses the following request header format: `Host: {app-name}.example.com`. Use the following cURL command to verify:
+* A CN *with a wildcard* (e.g. *.example.com) uses the following request header format: `Host: {app-name}.example.com`. Use the following cURL command to verify:
```

Comment:
@sundippatel FYI, out standard is now to only use bold for UI elements.

Reply:
Translators will not translate anything in bold.

Reply:
👍